### ISSUE TITLE
os.uname() not supported on Windows

### ISSUE THREAD

**Issue by jsikorsky** [vshymanskyy/blynk-library-python#27]
`os.uname()` function is not supported on Windows platform (I use it for development).
Since it's used only in banner upon library load, it's not crucial IMO and can be replaced by `sys.platform`, or omitted altogether.

Jurij

**Comment by vshymanskyy** [comment]
Does it crash? ;(

**Comment by jsikorsky** [comment]
Of course not, it just does not work without this simple fix.

Jurij

po 18. 2. 2019 v 20:09 odesílatel Volodymyr Shymanskyy <
notifications@github.com> napsal:

**Comment by jsikorsky** [comment]
Well, in wider context it actually does crash - `AttributeError` is thrown upon import.

```
Traceback (most recent call last):
  File "C:/Users/jurij/Documents/waveforms/python/blynk_test2.py", line 2, in <module>
    from BlynkLib import Blynk
  File "C:\Users\jurij\Documents\waveforms\python\BlynkLib\__init__.py", line 1, in <module>
    from .BlynkLib import Blynk
  File "C:\Users\jurij\Documents\waveforms\python\BlynkLib\BlynkLib.py", line 50, in <module>
    /___/ for Python v""" + _VERSION + " (" + os.uname()[1] + ")\n")
AttributeError: module 'os' has no attribute 'uname'
```

Regards,

Jurij

**Comment by vshymanskyy** [comment]
OK thanks, will fix asap

**Comment by jsikorsky** [comment]
propose fix #32

**Comment by jsikorsky** [comment]
propose fix #33

**Comment by parvaizahmad** [comment]
Use platform.uname() instead of os.uname().

**Comment by zurgeg** [comment]
i just proposed a fix at PR #42

**Comment by zurgeg** [comment]
This issue should be closed since PR #33 is merged.
